2001-3000/LIT2332serata.xml

```diff
@@ -1,10 +1,12 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT2332serata">
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Śǝrʿāta beta krǝstiyān</title>
+            <title xml:lang="gez" xml:id="t1">ስርዓተ፡ ቤት፡ ክርስቲያን፡</title>
+            <title xml:lang="gez" corresp="#t1" type="normalized">Śǝrʿāta beta krǝstiyān</title>
+            <title xml:lang="en" corresp="#t1">Structure of the church</title>
```

Comment:
Śǝrʿāta = Structure or Order?

Reply:
"Structure" may sound strange, but the texts concern indeed the material structure. "Structure" is possible for "ser'at", even though "order" is the first one think of but this translation is misleading.

Reply:
Well the title marked as xml:lang="en" corresp="#t1" should be the exact translation of Ethiopic; a paraphrase should be ideally encoded as t2 (unless Structure is a meaning registered by dictionaries)

Reply:
ok